spec/models/social_account_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
```

Comment:
Indentation of this file (and probably others as well) mismatch! Make sure to use 2 spaces for indentation. See Rubocop recommendations. :-)

Reply:
Solved via bca5892b05d79b03e617b1d8e77268e7d5dfd114